addons/mass_mailing_crm/tests/test_mass_mailing_crm.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import Form, TransactionCase
+
+
+class TestMassMailingCrm(TransactionCase):
+
+    def test_mass_mailing_values_for_leads(self):
```

Comment:
Would test with a test user that is both sales and marketing, just to try out ACLs . You may define a new user using the ``mail_new_test_user`` tool, allowing to quickly create users (with relevant context keys to avoid noise in test, ...).